habitat_sim/utils/__init__.py

```diff
@@ -9,4 +9,4 @@
 # fixed
 from habitat_sim.utils.common import quat_from_angle_axis, quat_rotate_vector
```

Comment:
Probably needs to have a `from habitat_sim.utils import common` added?

Reply:
I admit that, due to the work I did on all the python inspection, I have a deeper-than-necessary knowledge of the module internals and I'm quite sure it works this way too, since `common` get pulled in as a side effect of `from habitat_sim.utils.common import thing`.

Added that explicitly for clarity, thanks.